### PR TITLE
Feature/assingment notes with scores

### PR DESCRIPTION
**Pull request by melisabok**
Let's do a code review during the hangouts.

### CODE REVIEW

File: venues/auai.org/UAI/2018/python/match-reviewers.py [hunk 1]
```diff
@@ -88,14 +88,70 @@ def create_assignment_note(forum, label):
...
+
+def get_paper_scores(forum_id):
+    papers = [p for p in paper_metadata if p.forum == forum_id]
+
+    if papers:
+        return papers[0].content['groups'][configuration_note_params['content']['match_group']]
+    else:
+        return []
+
+def weigth_scores(group_scores, weigths):
```
Comment (mspector) [inline venues/auai.org/UAI/2018/python/match-reviewers.py]: small typo: weigth_scores --> weight_scores

Comment (melisabok) [reply venues/auai.org/UAI/2018/python/match-reviewers.py]: Thanks!

**Comment by mspector** [comment]
@melisabok should this be merged?

**Comment by melisabok** [comment]
Yes, the matching browser was deployed in live, we need this to run the matching

**Pull request merged**